run.py

```diff
@@ -109,6 +122,23 @@ def eval(args):
         path = f'{args.folder}/{args.name}-latents.png'
         viz_latents(model, data_loader, args.latents, path)
 
+    scores = {'fid_score': [], 'is_score': []}
+    fid_score = metrics.FID()
+    inception_score = metrics.InceptionMetric()
```

Comment:
To make this extra pretty, you could if you want try to create a flag --fid like I have for --sample and --latents